packages/framework-core/src/booster-command-dispatcher.ts

```diff
@@ -41,10 +42,17 @@ export class BoosterCommandDispatcher {
 
     const commandInstance = createInstance(commandClass, commandInput)
 
-    const register = new Register(commandEnvelope.requestID, commandEnvelope.currentUser, commandEnvelope.context)
+    const register = new Register(
+      commandEnvelope.requestID,
+      commandEnvelope.responseHeaders,
```

Comment:
If we remove the `responseHeaders` field from the `CommandEnvelope`, here we should set a reference to the `responseHeaders` object from the context:

```suggestion
      context.responseHeaders,
```